cincinnati/src/plugins/internal/cincinnati_graph_fetch.rs

```diff
@@ -260,6 +274,7 @@ mod tests {
     fetch_upstream_failure_test!(
         name: fetch_fail_invalid_url,
         upstream: "invalid.url",
+        gb_timeout: 10,
```

Comment:
I'm still not convinced even though the numbers are randomized now ;-) If we don't use them, let's skip them for now please. I would prefer to actually test the timeout of course!